### DISCUSSION TITLE
Still saying "mentioned this issue..." despite editing the comment

### DISCUSSION THREAD

**Discussion by gmoniava** [community/community#63067] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

1. Imagine you are in issue A, and make a comment there where you reference issue B.
2. You probably know that in this case in the issue B there will be text saying "user Joe mentioned this issue [10 hours ago]"
3. Now imagine I go back to issue A and edit the comment where I referenced issue B such that I delete out the reference to issue B.
4. On issue B the text "user Joe mentioned this issue [10 hours ago]" still remains.

Should it be like that? I also tested now and even if I delete the comment (Step 1), the text still remains (as in Step 4).

**Comment by lonnywong** [comment]
BTW, if I'm the owner of the repository, can I delete the mentions?
